### ISSUE TITLE
Missing funds

### ISSUE THREAD

**Issue by gcameron21** [bitpay/copay#10410]
Hi,

Looking for some help please.  To kick things off, I have cleared the cache, scanned address, uninstalled and restored with backup, and used the copay recovery tool. Still missing funds.   

Now to the issue,I use the bitpay wallet on my galaxy phone.  2 days ago I sent 3 separate transaction to my wallet for .08, .29, and .10. Giving me a balance of roughly .47.  

Everything was working great at first. I sent a few smaller transactions out, and was sitting around .330 left after a day. I sent out a .025 transaction yesterday to an address, and then I sent a repeat transaction to the same address for the same amount about 10 minutes later. After sliding the send bar, my screen just went right back to my dashboard home, which was abnormal. When I clicked on my wallet .29 was taken out of my balance, but there was no activity showing the withdrawl.  I went to the blockchain to research the address I had sent to hoping my wallet had taken the .29 to send the .025 transaction and it would send the rest back to my wallet like normal.  

To my surprise there was no transaction, I've kept checking but still nothing.  My activity still show the original deposit of .29 and the address that recieved the original .29 still show a balance of balance of .29.  

Confused I then sent a test transaction to that address with the .29 to see what it would do. It sent it out and shows moved in my "activity" but didnt show up in my balance.  It's like my wallet somehow disconnected  itself from the original deposit address and it's not including as part of my wallet anymore.  

To sum it up, I'm short. 29 btc and the .025 transaction was never sent.  I have attached screen shots showing my activity chain, if you look and add it up you will see there is .29 unaccounted for.  Any advice? 

![Screenshot_20191209-100349_BitPay](https://user-images.githubusercontent.com/58702832/70460805-68807900-1a74-11ea-88ba-7c419944a89f.jpg)
![Screenshot_20191209-100359_BitPay](https://user-images.githubusercontent.com/58702832/70460829-73d3a480-1a74-11ea-92d3-90f21425c1b3.jpg)
![Screenshot_20191209-100406_BitPay](https://user-images.githubusercontent.com/58702832/70460839-7635fe80-1a74-11ea-9c38-61d55e9852ff.jpg)

**Comment by BGalliotti** [comment]
Hello,

Thank you for reaching out to us.

Please write to the BitPay Customer Success team: https://help.bitpay.com/requestHelp providing your wallet ID.

Their representatives can address your inquiry there.

We appreciate your patience.

Kind regards.

**Comment by walletrescue** [comment]
@gcameron21 
is this issue still with you
i had the same issue with my customer
I can help with your problem
 : anywalletrecovery@gmail.com
